src/Dotnet.Script/Program.cs

```diff
@@ -267,21 +282,19 @@ private static async Task<int> RunScript(string file, bool debugMode, LogFactory
             var absoluteFilePath = Path.IsPathRooted(file) ? file : Path.Combine(Directory.GetCurrentDirectory(), file);
             var directory = Path.GetDirectoryName(absoluteFilePath);
 
-            using (var filestream = new FileStream(absoluteFilePath, FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            var rawCode = File.ReadAllText(absoluteFilePath);
+            var sourceText = SourceText.From(RemoveShebang(rawCode));
```

Comment:
Could this cause some issues with breakpoints since we alter the original "source" ?

Reply:
hmmm...great question, I will explore

Reply:
What happens if we just leave it as is? We used to have `#! "netcoreapp2.1"` and the compiler does not care about that?

Reply:
Here's the funky thing, if you have JUST the shebang the compiler don't care and it works fine.
 
If you have shebang AND the #! "netcoreapp2.1" then the compiler barfs on the shebang. 
I just updated it so that the shebang is replaced and extra CR is added so line numbers remain the same.

Reply:
Guess we don't have to touch the source if we stick we just one shebang directive 👍